src/main/webapp/layout/lemmaKorr.css

```diff
@@ -0,0 +1,98 @@
+#list {
```

Comment:
Some of these entries should also be changed. They sound rather generic and could cause conflicts with the regular layout.

IDs like e.g. "#list" should be prefixed
`a` will override all hyperlinks on the whole page (even if the CSS file is only included in the admin servlet right now, e.g. also in the menu etc.), so the hyperlinks in the view should use a separate class (maybe also with a prefix) which the CSS statements can override precisely.